static-site/src/components/nav-bar/index.tsx

```diff
@@ -98,23 +98,30 @@ const NavBar = ({ minified }: {
   }
 
   const LogoType = () => {
-    const title = "Nextstrain";
-    const rainbowTitle = title.split("").map((letter, i) =>
-      <NavLogoCharacter key={i} $colorIndex={i}>{letter}</NavLogoCharacter>
-    );
-    const SubTitle = styled.div`
-      color: black;
-    `;
-    return (
-      minified ?
-        <div/>
-        :
-        <a href="/">
-          {rainbowTitle}
-          {groupsApp &&
-            <SubTitle>Groups Server</SubTitle>}
-        </a>
-    );
+		const title = "Nextstrain";
+		const rainbowTitle = title.split("").map((letter, i) => (
+			<NavLogoCharacter key={i} $colorIndex={i}>
+				{letter}
+			</NavLogoCharacter>
+		));
+		const SubTitle = styled.div`
+			color: black;
+		`;
+		/* Hide the title on mobile so the LOGIN link is visible */
+		/* https://github.com/nextstrain/nextstrain.org/issues/962 */
+		const ResponsiveTitle = styled.span`
+			@media (max-width: 768px) {
+				display: none;
+			}
+		`;
+		return minified ? (
+			<div />
+		) : (
+			<a href="/">
+				<ResponsiveTitle>{rainbowTitle}</ResponsiveTitle>
+				{groupsApp && <SubTitle>Groups Server</SubTitle>}
```

Comment:
If we're going to hide the title, we want to hide all of it: 

```suggestion
				<ResponsiveTitle>
				  {rainbowTitle}
  				{groupsApp && <SubTitle>Groups Server</SubTitle>}
  			</ResponsiveTitle>
```

note: this may require changing L112 to be `styled.div` instead of the current `styled.span`.